master.js

```diff
@@ -17,10 +22,25 @@ var _gaq = [
 
 window.test = function(expression) {
   var result = (typeof expression === "string" ? expression : !!expression ? 'Yes' : 'No');
-  document.write('<td class="' + result.toLowerCase() + ' current">' + result + '</td><td></td>');
+
+  // Last SCRIPT in the document is the one currently executing
+  var thisScript = document.currentScript || document.scripts[document.scripts.length-1];
+
+  var td = thisScript.parentNode;
+  var tr = td.parentNode;
+
+  //document.write('<td class="' + result.toLowerCase() + ' current">' + result + '</td><td></td>');
+
+  // Insert the two new TD elements after the one that contains the SCRIPT
+  td = tr.insertBefore(document.createElement("td"), td.nextSibling);
+  td.className = result.toLowerCase() + " current";
+  td.textContent = td.innerText = result;
+
+  tr.insertBefore(document.createElement("td"), td.nextSibling);
 };
 
-document.write('<style>td:nth-of-type(2) { outline: #aaf solid 3px; }</style>');
+// Moved to 'master.css'
+//document.write('<style>td:nth-of-type(2) { outline: #aaf solid 3px; }</style>');
```

Comment:
also, is there any reason to keep this?

Reply:
Not much of a reason to keep those comments. I'll go ahead and drop them.